builder/src/upload.js

```diff
@@ -0,0 +1,65 @@
+const rp = require('request-promise')
+const fs = require('fs')
+const glob = require('glob')
+
+class Upload {
+  constructor (config) {
+    this.config = config
+  }
+
+  findPackages (outputDir) {
+    try {
+      return glob.sync(outputDir + '**/*.nest', {})
```

Comment:
For this to be relevant, you have to clean this directory before building anything (or you'll pick up the results of the previous one).

Is it done? I can't find any trace of that in the codebase, but I looked quickly so I might miss it

Reply:
`outputDir` is defined in the `receiver.js` file as the current timestamp:

```javascript
async buildOneManifest (buildId, manifests, exitCode) {
  /* */
  const outputDir = this.config.output_dir + (new Date()).getTime() + '/'
  /* */
  child.on('exit', async (code) => {
    const res = await this.upload.toNestServer(outputDir)
    /* */
  })
} 
```

So each manifest is compiled in his own folder

Reply:
That makes sense